app/controllers/support_ticket/base_controller.rb

```diff
@@ -10,4 +10,12 @@ def parent_support
   def thank_you
     render 'support_tickets/thank_you'
   end
+
+private
+
+  def redirect_if_no_form_data_exists
+    return unless request.get?
```

Comment:
Are non-GET requests really excluded here? If they are doing `before_action :require_support_ticket_data!, only: :new` might be a little safer.